coronaqr.go

```diff
@@ -73,6 +73,7 @@ type TestRecord struct {
 	//   "description": "Type of Test",
```

Comment:
Here and below, you can remove these comments now that you have added the fields